apps-rendering/package.json

```
@@ -8,19 +8,18 @@
 		"ios >= 12"
 	],
 	"scripts": {
-		"preinstall": "../scripts/preinstall.mjs",
 		"upload": "node-riffraff-artifact",
 		"copy-manifest": "cp dist/assets/manifest.json dist/server/manifest.json",
 		"copy-fonts": "cp -R assets/fonts dist/assets/fonts",
 		"test": "jest --config config/jest.config.js --verbose",
-		"test:coverage": "yarn test; open coverage/index.html",
+		"test:coverage": "pnpm test; open coverage/index.html",
 		"test:watch": "jest --config config/jest.config.js --watch",
 		"lint": "eslint -c config/.eslintrc.js ./ --ext '.ts,.tsx' --cache",
 		"lint:fix": "eslint -c config/.eslintrc.js ./ --ext '.ts,.tsx' --fix --cache",
 		"watch:server": "webpack --config webpack.config.ts --config-name server --env watch",
 		"watch:client": "webpack serve --config webpack.config.ts --config-name client",
 		"watch:cdk": "tsc -w -p config/tsconfig.cdk.json",
-		"watch": "yarn watch:server & yarn watch:client",
+		"watch": "pnpm '/^watch:(?!cdk$).*/'",
```

Closer to the original:

```suggestion
		"watch": "pnpm '/^watch:(client|server)/'",
```

---

yeah much nicer